internal/config/config.go

```diff
@@ -29,7 +29,6 @@ type Config struct {
 	GitlabWebhookSecretKey string
 	Services               map[string]Service
 	Tenants                map[string]Tenant
-	Debug                  bool
```

Comment:
I think we should keep this, we may want to utilize it in the future

Reply:
Added that SKIP_WEBHOOK_VALIDATION var :)